developer/src/tike/compile/CompileKeymanWeb.pas

```diff
@@ -1742,8 +1742,11 @@ function TCompileKeymanWeb.ValidateLayoutFile(var sLayoutFile: string; const sVK
       if FValid = Key_Invalid then
       begin
         ReportError(0, CERR_TouchLayoutInvalidIdentifier, 'Key "'+FId+'" on "'+FPlatform.Name+'", layer "'+FLayer.Id+'" has an invalid identifier.');
+      end
+      else if (FValid = Key_Unicode_Multi) and not IsKeyboardVersion15OrLater then
```

Comment:
I have checked and the value `Key_Unicode` is not used anywhere, so it's safe to split it into `Key_Unicode` and `Key_Unicode_Multi`.